dvc/render/convert.py

```diff
@@ -29,10 +29,8 @@ def _group_by_rev(datapoints):
 
 
 def to_json(renderer, split: bool = False) -> List[Dict]:
-    from copy import deepcopy
-
     if renderer.TYPE == "vega":
-        grouped = _group_by_rev(deepcopy(renderer.datapoints))
+        grouped = _group_by_rev(renderer.datapoints)
```

Comment:
Seems like we never mutate `renderer.datapoints`, not in `_group_by_rev`, not in `to_json` or outside. So I don't think we need deepcopy here unless I am missing something.

Reply:
There is also no operation after `to_json` (https://github.com/iterative/dvc/blob/cbe46cccd860f8cd7ba447d89700c26eb0935a11/dvc/commands/plots.py#L137-L138), so apparently it would not matter even if we mutate them